addons/account/models/account.py

```diff
@@ -659,6 +659,9 @@ def compute_all(self, price_unit, currency=None, quantity=1.0, product=None, par
                 'analytic': boolean,
             }]
         } """
+        # TODO master: put this into method signature, it was hided into context because of api stable policy
+        basevalues = self.context.get('basevalues', false)
```

Comment:
Suggesting and correcting to ```base_vals = self._context.get('base_vals')``` - no need to give ```False``` as fallback as ```None``` will be sufficient. Variable ```basevalues``` is not soo pretty imo. 
Besides, I doubt that self.context is working...it must be self.env.context and ```False``` instead of false.